src_kolko_krzyżyk/TicTacToe_expand.py

```diff
@@ -185,7 +185,35 @@ def main(self):
             else:
                 player = 0
 
-game = TicTacToe(5)
+def menu():
+    print('''\n\n\t"Simple game for two person in Tic Tac Toe.
+     Player decides how big the board is. Valid board (2 to n)
+     where board are matrix n x n. Good Luck and Have Fun."\n\n''')
+    # default matrix_choice
+    matrix_choice = 3
+    while True:
```

Comment:
You can move the logic of asking the user of choice to different function which could looks like that:
```python
    matrix_size = set_matrix_size()
```
where the content of this function could be:
```
def set_matric_size():
    while True:
        value = input(...)
        try:
            return int(value)
        except (TypeError, ValueError):
            print(...)
```